### PR TITLE
[CORE-373] add ordering SYNCORE-717

### PR DESCRIPTION
**Pull request by Zhebr**

### CODE REVIEW

File: syncano/models/manager.py [hunk 1]
```diff
@@ -388,19 +388,34 @@ def limit(self, value):
         return self
 
     @clone
-    def order_by(self, field):
+    def ordering(self, order='asc'):
```
Comment (23doors) [inline syncano/models/manager.py]: It's either this or that.

DO manager should only have: "order_by" that sets query['order_by'] = field or -field if order=desc. It should not pass 'ordering' to query.
Other managers should only have "ordering" that sets query['ordering'] = asc/desc.

File: syncano/models/manager.py [hunk 2]
```diff
@@ -711,6 +711,28 @@ def exclude(self, *args):
         self.endpoint = 'list'
         return self
 
+    def ordering(self, order=None):
+        pass
```
Comment (23doors) [inline syncano/models/manager.py]: I think we should raise an exception here. Saying that user should use order_by. Silently failing may be tough to debug.

Comment (Zhebr) [inline syncano/models/manager.py]: agree

**Comment by 23doors** [comment]
lgtm!

**Pull request merged**